applications/grandpa-tla.md

```diff
@@ -0,0 +1,168 @@
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Alfredo Garcia
+
+### Contact
+
+- **Contact Name:** Alfredo Garcia
+- **Contact Email:** oxarbitrage@gmail.com
+- **Website:** https://github.com/oxarbitrage
+
+### Legal Structure
+
+- **Registered Address:** 
+- **Registered Legal Entity:** Alfredo Garcia
+
+### Team's experience
+
+I am an open-source developer actively involved in several blockchain projects. Currently, I work at the Zcash Foundation as a Rust engineer, where I have deepened my understanding of cryptography and software design principles. This role allows me to apply my skills effectively in developing secure and efficient protocols.
+
+### Team Code Repos
+
+The following are experimental specifications I have developed for educational purposes, all written in PlusCal:
+
+- [Zcash P2P Specification](https://github.com/oxarbitrage/zcash-p2p-spec)
+- [Sinsemilla Specification](https://github.com/oxarbitrage/sinsemilla/tree/main/spec)
+- [Zebra gRPC Scan Specification - gRPC Overview](https://github.com/oxarbitrage/zebra-grpc-scan-spec/blob/main/grpc.md)
+- [Zebra gRPC Scan Specification - Client Integration](https://github.com/oxarbitrage/zebra-grpc-scan-spec/blob/main/client_integration.md)
+
+## Development Status :open_book:
+
+Research stage. Development not started.
+
+## Development Roadmap :nut_and_bolt:
+
+Based on the [deliverables section of the RFP](https://grants.web3.foundation/docs/RFPs/formal_guarantees_for_grandpa#deliverables) but splitting it out into 3 milestones.
+
+### Overview
+
+- **Total Estimated Duration:** 5 months
+- **Full-Time Equivalent (FTE):**  0.5 FTE
+- **Total Costs:** 30,000 USD
+- **DOT %:** Percentage of Total Costs to be paid in (vested) DOT (≥ 50%)
+
+### Milestone 1 — Basic functionality
+
+- **Estimated duration:** 2 months
+- **FTE:**  0,5
+- **Costs:** 12,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / MIT |
+| **0b.** | Proof Artifact | Models and specs in TLA+ that adhere to protocol implementations with reasonable abstraction gaps. As a stepping stone, the block production mechanism could be instantiated with PoW or BABE. |
+| **0c.** | Documentation | A document describing the design decisions in the modeling process, including justification for abstractions and assumptions (e.g. on the network model, latency, behavior of validators, nature of faults) w.r.t to protocol in the Paper and Spec. In-line comments in the TLA+/ PlusCal models that provide a clear mapping to the feature/property. |
+| **0d.** | Engagements | Engage with the Web3 research team via regular meetings to refine the models and specs. For eg., clarify any assume/ rely reasoning made in the protocols. Engage with Web3 team to determine if detected bugs's are spurious. |
```

Comment:
Can you please specify what exactly is being delivered here? I suggest something like "Protocol of conversations with stakeholders."

Reply:
Hey, thanks for the feedback. I am out of the office for this week, ill be  making all proposed changes after i get back if that is ok with you.

The `Engagements` deliverable is part of https://grants.web3.foundation/docs/RFPs/formal_guarantees_for_grandpa#deliverables and i understand it as regular meetings with the Web3 researchers to discuss progress, feedback, bugs and such for the overall project while it is going. That is why i added it to 2 milestones.

Happy to change that.

Reply:
Thanks for the quick reply. I didn't notice it was part of the RFP.